docs/other-topics.md

```diff
@@ -63,11 +63,11 @@ echo $path; // "/path/to/subdir/blog/88"
 ?>
 ```
 
-## Usage in a View Helper
+## Route Helpers
 
-You may wish to be able to generate routes from your views. A generic view helper _Helper\Url_ is available to assist you in creating these urls.
+You may wish to be able to generate routes from your views. _Helper\Route_ and Helper\RouteRaw are available to assist you in creating these urls.
```

Comment:
@pmjones I guess you may be working on docs, so please note the missing `_ _` in `Helper\RouteRaw`